libs/template/renderer.go

```diff
@@ -127,10 +127,16 @@ func (r *renderer) computeFile(relPathTemplate string) (file, error) {
 	// If file name does not specify the `.tmpl` extension, then it is copied
 	// over as is, without treating it as a template
 	if !strings.HasSuffix(relPathTemplate, templateExtension) {
+		// Execute relative path template to get destination path for the file
+		relPath, err := r.executeTemplate(relPathTemplate)
```

Comment:
I'm not entirely sure why this works. My understanding is that `r.executeTemplate` returns the materialised content of the provided template. Why does it destination path in this case?

Reply:
The execute template function takes in a template string, and returns the output after executing it. It's unaware of where it's a file path or the content of a file. 

The path to the file forms the template that we use to compute the path for project file